lib/LedgerSMB/Scripts/payment.pm

```diff
@@ -482,14 +482,14 @@ sub display_payments {
                 $payment->{"paid_$_->{contact_id}"} = ''
                     unless defined $payment->{"paid_$_->{contact_id}"};
             }
-            $invoice->[6] = $invoice->[3] - $invoice->[4] - $invoice->[5];
-            $contact_to_pay += $invoice->[6];
-            $invoice->[7] = $invoice->[6]->to_db;
+            $invoice->[6] = $invoice->[3] - $invoice->[4] - $invoice->[5];  ## no critic (ProhibitMagicNumbers) sniff
+            $contact_to_pay += $invoice->[6];  ## no critic (ProhibitMagicNumbers) sniff
+            $invoice->[7] = $invoice->[6]->to_db;  ## no critic (ProhibitMagicNumbers) sniff
```

Comment:
These all would benefit from a series of constants "naming" the column indexes.

Reply:
I will pass on this task as I have no idea of the namings.

Reply:
ok. then I'll do that as soon as this gets merged.